packages/components/breadcrumbs/breadcrumbs.spec.ts

```diff
@@ -35,15 +34,18 @@ function findAllCustomSeparators(debugElement: DebugElement): DebugElement[] {
     return debugElement.queryAll(By.css('.custom-separator'));
 }
 
+const customBreadcrumbsProvider = provideKbqBreadcrumbsConfiguration({
```

Comment:
сейчас только заметил, что утилита по созданию провайдера называется иначе чем другие подобные, мб приведем к общему виду? 
```suggestion
const customBreadcrumbsProvider = kbqBreadcrumbsConfigurationProvider({
```